tekton/README.md

```diff
@@ -0,0 +1,118 @@
+
+## Create an official release
+
+Official releases are performed from [the `prow` cluster](https://github.com/tektoncd/plumbing#prow)
+[in the `tekton-releases` GCP project](https://github.com/tektoncd/plumbing/blob/master/gcp.md).
+This cluster
+[already has the correct version of Tekton installed](#install-tekton).
+
+To make a new release:
+
+1. (Optionally) [Apply the latest versions of the Tasks + Pipelines](#setup)
+2. (If you haven't already) [Install `tkn`](https://github.com/tektoncd/cli#installing-tkn)
+2. [Run the Pipeline](#run-the-pipeline)
+3. Create the new tag and release in GitHub
+   ([see one of way of doing that here](https://github.com/tektoncd/pipeline/issues/530#issuecomment-477409459)).
+   _TODO(tektoncd/pipeline#530): Automate as much of this as possible with Tekton._
+4. Add an entry to [the README](../README.md) at `HEAD` for docs and examples for the new release
+   ([README.md#read-the-docs](README.md#read-the-docs)).
+5. Update the new release in GitHub with the same links to the docs and examples, see
+   [v0.1.0](https://github.com/tektoncd/pipeline/releases/tag/v0.1.0) for example.
+
+### Run the Pipeline
+
+To use [`tkn`](https://github.com/tektoncd/cli) to run the `publish-tekton-pipelines` `Task` and create a release:
+
+1. Pick the revision you want to release and update the
+   [`resources.yaml`](./resources.yaml) file to add a
+   `PipelineResoruce` for it, e.g.:
+
+   ```yaml
+   apiVersion: tekton.dev/v1alpha1
+   kind: PipelineResource
+   metadata:
+     name: tekton-triggers-vX-Y-
+   spec:
+     type: git
+     params:
+     - name: url
+       value: https://github.com/tektoncd/triggers
+     - name: revision
+       value: vX.Y.Z-invalid-tags-boouuhhh # REPLACE with the commit you'd like to build from
+   ```
+  
+3. To run against your own infrastructure (if you are running
+   [in the production cluster](https://github.com/tektoncd/plumbing#prow) the default account should
+   already have these creds, this is just a bonus - plus `release-right-meow` might already exist in the
+   cluster!), also setup the required credentials for the `release-right-meow` service account, either:
+
+   - For
+     [the GCP service account `release-right-meow@tekton-releases.iam.gserviceaccount.com`](#production-service-account)
+     which has the proper authorization to release the images and yamls in
+     [our `tekton-releases` GCP project](https://github.com/tektoncd/plumbing#prow)
+   - For
+     [your own GCP service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+     if running against your own infrastructure
+
+
+2. [Connect to the production cluster](https://github.com/tektoncd/plumbing#prow):
+
+  ```bash
+  gcloud container clusters get-credentials prow --zone us-central1-a --project tekton-releases
+  ```
+
+6. Run the `release-pipeline` (assuming you are using the production cluster and
```

Comment:
The numbering here is off as well as in the `Tekton` repo. I made a small PR: https://github.com/tektoncd/pipeline/pull/1321. Could we get this merged and then use this here?

Reply:
done!

Reply:
Still see the numbering issue in [this file](https://github.com/tektoncd/triggers/blob/5f4c6cfc095264464310ca01658437c603769a14/tekton/README.md#run-the-pipeline). The 4th point needs to be intended more due to the code fence I believe.

Reply:
yeah...should be fixed now